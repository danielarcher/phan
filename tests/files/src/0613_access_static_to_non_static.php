<?php 

class PrimaryClass
{
    public static function staticFunction()
    {
        return true;
    }
}

class ChildClass extends PrimaryClass
{
    public function staticFunction()
    {
        return false;
    }
}
