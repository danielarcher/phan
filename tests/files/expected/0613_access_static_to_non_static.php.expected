%s:13 PhanAccessStaticToNonStatic Cannot make static method \PrimaryClass::staticFunction() non static
